temporal-sdk/src/main/java/io/temporal/internal/statemachines/VersionStateMachine.java

```diff
@@ -36,20 +36,27 @@
 import java.util.Map;
 import java.util.Objects;
 import java.util.Optional;
+import javax.annotation.Nullable;
 
 final class VersionStateMachine {
 
   static final String MARKER_VERSION_KEY = "version";
   static final String MARKER_CHANGE_ID_KEY = "changeId";
   static final String VERSION_MARKER_NAME = "Version";
 
+  private static final Command FAKE_COMMAND =
+      Command.newBuilder()
+          .setCommandType(CommandType.COMMAND_TYPE_RECORD_MARKER)
+          .setRecordMarkerCommandAttributes(RecordMarkerCommandAttributes.getDefaultInstance())
+          .build();
+
```

Comment:
[nit] This nice bit of cleanup you did here and in the MSE state machine - it maybe would be nice to just have a common helper function that accepts `Optional<RecordMarkerCommandAttributes>` and constructs with default if empty, otherwise passed in args.

Would get rid of the command builder preamble in both places